app/assets/javascripts/_stick-at-top-when-scrolling.js

```diff
@@ -0,0 +1,19 @@
+(function(GOVUK, GDM) {
+
+  GOVUK.GDM = GDM;
```

Comment:
What does GDM stand for?  Guessing, something Digital marketplace?  I'm just curious, don't remember it.